code/game/objects/items/AI_modules/_AI_modules.dm

```diff
@@ -30,22 +30,28 @@
 
 /obj/item/ai_module/examine(mob/user as mob)
 	. = ..()
-	if(Adjacent(user))
```

Comment:
I'd probably keep this in by doing:

```
if((isliving(user) && Adjacent(user)) || isobserver(user)
```

Reply:
Also link this issue to #68604 to properly close it since the underlying issue was never resolved.

Reply:
Isn't this just for the Module Board that contains the laws?

Reply:
https://github.com/tgstation/tgstation/blob/fc91c2ed1f9e2311382472acb746feb69cdcca68/code/modules/mob/living/silicon/examine.dm#L1-L6

Damn, nevermind you are right.  I should have looked closer but I assumed the underlying issue was the same since it was both dealing with examine and laws.  

I'll reclose the issue.